lib/elixir/lib/string.ex

```diff
@@ -1755,8 +1755,8 @@ defmodule String do
     end
   end
 
-  @compile {:inline, decompose: 1}
-  defp decompose(string) do
+  @compile {:inline, dissect: 1}
+  defp dissect(string) do
```

Comment:
Renamed to `chars_and_length` as its second part is an equivalent of `String.length/1`.